### PR TITLE
Add GTC 2019 video and presentation do the documentation

### PR DESCRIPTION
**Pull request by JanuszL**
- adds a link to the presentation and video about DALI in GTC19
- adds a link to the presentation and video about DALI and TRT on Xavier
  in GTC19
- adds a link to the developer page
- adds a link to the blog post

Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

### CODE REVIEW

**Comment by JanuszL** [comment]
!build

File: README.rst [hunk 1]
```diff
@@ -459,6 +459,10 @@ Additional resources
 --------------------
 
 - GPU Technology Conference 2018 presentation about DALI, T. Gale, S. Layton and P. Tredak: `slides <http://on-demand.gputechconf.com/gtc/2018/presentation/s8906-fast-data-pipelines-for-deep-learning-training.pdf>`_, `recording <http://on-demand.gputechconf.com/gtc/2018/video/S8906/>`_.
+- GPU Technology Conference 2019 presentation about DALI, J.Lisiecki and Michał Zientkiewicz: `slides <https://developer.download.nvidia.com/video/gputechconf/gtc/2019/presentation/s9925-fast-ai-data-pre-processing-with-nvidia-dali.pdf>`_, `recording <https://developer.nvidia.com/gtc/2019/video/S9925/video>`_.
+- GPU Technology Conference 2019 presentation about DALI adn TensorRT on Xavier: `slides <https://developer.download.nvidia.com/video/gputechconf/gtc/2019/presentation/s9818-integration-of-tensorrt-with-dali-on-xavier.pdf>`_, `recording <https://developer.nvidia.com/gtc/2019/video/S9818/video>`_.
```
Comment (klecki) [inline README.rst]:
```suggestion
- GPU Technology Conference 2019 presentation about DALI and TensorRT on Xavier: `slides <https://developer.download.nvidia.com/video/gputechconf/gtc/2019/presentation/s9818-integration-of-tensorrt-with-dali-on-xavier.pdf>`_, `recording <https://developer.nvidia.com/gtc/2019/video/S9818/video>`_.
```

File: README.rst [hunk 2]
```diff
@@ -459,6 +459,10 @@ Additional resources
 --------------------
 
 - GPU Technology Conference 2018 presentation about DALI, T. Gale, S. Layton and P. Tredak: `slides <http://on-demand.gputechconf.com/gtc/2018/presentation/s8906-fast-data-pipelines-for-deep-learning-training.pdf>`_, `recording <http://on-demand.gputechconf.com/gtc/2018/video/S8906/>`_.
+- GPU Technology Conference 2019 presentation about DALI, J.Lisiecki and Michał Zientkiewicz: `slides <https://developer.download.nvidia.com/video/gputechconf/gtc/2019/presentation/s9925-fast-ai-data-pre-processing-with-nvidia-dali.pdf>`_, `recording <https://developer.nvidia.com/gtc/2019/video/S9925/video>`_.
```
Comment (klecki) [inline README.rst]: I would suggest adding the titles of the presentations, to be less generic.

Comment (JanuszL) [reply README.rst]: Done

File: README.rst (hunk 1 above)
Comment (JanuszL) [reply README.rst]: Done

**Pull request merged**